method/message/messageGeneral.json

```diff
@@ -0,0 +1,41 @@
+{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id": "http://pop2020.dedis.ch/method/message/messageGeneral.json",
+	"title": "This represenets a message someone would want to publish on a specific channel",
+	"description": "Match general message content (Create LAO, Update LAO, Broadcast LAO, Create Meeting, Broadcast Meeting)",
+	"type": "object",
+
+	"properties": {
+		"data": {
+			"description": "[String] data contained in the message",
+			"$ref": "utility/base64String.json",
+			"$comment": "Note: the string is encoded in Base64"
+		},
+		"sender": {
+			"description": "[String] public key of the sender/organizer/server",
+			"$ref": "utility/base64String.json",
+			"$comment": "Note: the string is encoded in Base64"
+		},
+		"signature": {
+			"description": "[String] organizer's signature on data : Sign(Hash(data))",
+			"$ref": "utility/base64String.json",
+			"$comment": "Note: the string is encoded in Base64"
+		},
+		"message_id": {
+			"description": "[String] message id : Hash(data||signature)",
+			"$ref": "utility/base64String.json",
+			"$comment": "Note: the string is encoded in Base64"
+		},
```

Comment:
You can avoid the external ref just by using:
https://json-schema.org/understanding-json-schema/reference/non_json_data.html

Reply:
Indeed, it seems overkill! I will change it now!